wire-ios/Wire-iOS/Sources/UserInterface/Conversation Options/UIAlertController+ConversationGuestOptions.swift

```diff
@@ -74,4 +81,26 @@ extension UIAlertController {
                                                    confirmAction: confirmAction,
                                                    completion: completion)
     }
+
+    static func guestLinkTypeController(
+        completion: @escaping (GuestLinkType) -> Void
+    ) -> UIAlertController {
+        let controller = UIAlertController(title: nil, message: nil, preferredStyle: .actionSheet)
+
+        let optionOneAction = UIAlertAction(title: GuestRoom.Create.LinkWithPassword.action, style: .default) { _ in
+            completion(.secure)
+        }
+
+        let optionTwoAction = UIAlertAction(title: GuestRoom.Create.LinkWithoutPassword.action, style: .default) { _ in
```

Comment:
I would give more meaningful names for these actions, not just `optionOneAction`, `optionTwoAction`

Reply:
Done!